src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/SecretBuildWrapper.java

```diff
@@ -71,6 +87,22 @@
             }
         };
     }
+    
+    @Override
+    public OutputStream decorateLogger(AbstractBuild build, OutputStream logger)
+    		throws IOException, InterruptedException, RunnerAbortedException {
+    	if(showBindings)
+    		return super.decorateLogger(build, logger);
+    	
+    	Map<String,String> overrides = new HashMap<String,String>();
+        List<MultiBinding.Unbinder> unbinders = new ArrayList<MultiBinding.Unbinder>();
+        for (MultiBinding<?> binding : bindings) {
+            MultiBinding.MultiEnvironment environment = binding.bind(build, build.getWorkspace(), null, null);
```

Comment:
Do not call `bind` as it will actually do things, like copy files, which `setUp` already did. Just call `binding.variables()`, as `makeSensitiveBuildVariables` does.